### PR TITLE
/users/stats api endpoint documentation.

### PR DESCRIPTION
**Pull request by lisakycho**
New api endpoint was added to get the count of all system users.
https://github.com/mattermost/mattermost-server/pull/8847

### CODE REVIEW

File: v4/source/users.yaml [hunk 1]
```diff
@@ -305,6 +305,32 @@
...
           $ref: '#/responses/Forbidden'
 
+  /users/stats:
+    get:
+      tags:
+        - users
+      summary: Get total count of users in the system
+      description: |
+        Get a total count of users in the system.
+      parameters:
```
Comment (saturninoabril) [inline v4/source/users.yaml]: There's no `parameters` so you may omit this.

**Comment by cpanato** [comment]
Hello @lisakycho when do you have a chance can you please check the comment and fix the build issue?

thanks for this PR to improve the documentation

**Comment by hanzei** [comment]
@cpanato Would it be appreciated if I finish this PR?

**Comment by grundleborg** [comment]
@hanzei yes indeed! Please do :)

**Comment by hanzei** [comment]
This is ready for review

**Comment by cpanato** [comment]
@hanzei thanks! 😄

**Pull request merged**